fla/ops/path_attn/parallel.py

```
@@ -79,24 +82,15 @@
     @input_guard
     @autocast_custom_bwd
     def backward(ctx, do, dk_new):
-        q, k, v, w, g_cumsum, o, beta, L = ctx.saved_tensors
-        BT = 64
+        if do.shape[-1] > 64 and check_shared_mem('hopper') is False:
+            assert False, "Head dimension 128 only supported on Hopper or later. Stay tuned for Ampere support!"
```

Ruff (0.11.9)

Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)